Directory.Build.props

```diff
@@ -47,6 +47,16 @@
     <CoreLibSharedDir>$([MSBuild]::NormalizeDirectory('$(LibrariesProjectRoot)', 'System.Private.CoreLib', 'src'))</CoreLibSharedDir>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetsMobile)' == 'true'">
```

Comment:
nit: we could split this so AppleAppBuilder is built only for iOS/tvOS and AndroidAppBuilder for Android

Reply:
This condition controls which properties are defined. The runners are already only built on matching target OSs